src/components/WeightLossProgram/Testimony.tsx

```
@@ -0,0 +1,27 @@
+import React from 'react';
+
+type Props = {
+  imgSrc: string;
+  name: string;
+  testimony: string;
+};
+
+const Testimony = ({ imgSrc, name, testimony }: Props): JSX.Element => {
+  return (
+    <div className="flex flex-col items-center overflow-y-auto h-[450px]">
+      <figure className="w-[300px] flex flex-col items-center">
+        <img
+          className="h-[250px] w-[250px] rounded-full"
+          src={imgSrc}
+          alt={`${name}`}
+        />
+        <figcaption className="p-3 text-xl italic flex flex-col">
+          {name}
+          {testimony && <p>{testimony}</p>}
```

Don't put `<p>` tag inside of `<figcaption>`, rather use `<span>` or other inline HTML element.

---

Ok ok, I will fix it.
Could you please explain me why <p> is not good this time?

Thanks,
Fernando

---

It's not good nesting `block` elements inside of each other. For an example, if you nest it like this:
```html
<p>Text<p> is awesome</p></p>
```
React can ignore this error while building, but you can see it in browser when you click on Inspect Element. That's why its recommended to have one block element (specifically for text manipulation, this doesn't include `<div>`), and many `inline-block` elements like this:
```html
<p>Text<span> is awesome</span></p>
```
or like this:
```html
<figcaption>Text<span> is awesome</span></figcaption>
```